cloudlift/deployment/service_template_generator.py

```diff
@@ -280,7 +284,20 @@ def _add_service(self, service_name, config):
                         Principal=Principal("Service", ["ecs-tasks.amazonaws.com"])
                     )
                 ]
-            )
+            ),
+            Policies=[
+                Policy(
+                    PolicyName=service_name + "-Firelens-" + hashlib.sha1(service_name.encode()).hexdigest()[0:5],
+                    PolicyDocument=PolicyDocument(
+                        Statement=[
+                            Statement(
+                                Effect=Allow,
+                                Action=[PutRecordBatch, AssumeRole],
+                                Resource=["*"]
```

Comment:
Review to have more limited scope

Reply:
Action `PutRecordBatch` has been removed. Changes have been made, from now on, the statement would look like this:

```py
Statement(
      Effect=Allow,
      Action=[AssumeRole],
      Resource=[assume_role_resource or "*"],
  )
# where assume_role_resource is the default role assume role ARN that has been on the environment configuration
```